### ISSUE TITLE
Feature Request: List/Dict of Import Statements

### ISSUE THREAD

**Issue by Jeeppler** [Nuitka/Nuitka#191]
I am currently writing a small command line (cli) tool. This tool uses one direct dependency ([Chevron](https://pypi.org/project/chevron/)) which in turn depends on another dependency ([PyYAML](https://pypi.org/project/PyYAML/)). When I try to compile it I get the following error message:

```
$ python3 -m nuitka --follow-imports mycli.py 
Nuitka:WARNING:/usr/local/lib/python3.6/dist-packages/chevron/renderer.py:12: Cannot find 'tokenizer' in package 'chevron' as absolute import (tried tokenizer).
Nuitka:WARNING:/usr/local/lib/python3.6/dist-packages/chevron/main.py:15: Cannot find 'renderer' in package 'chevron' as absolute import (tried renderer).
Nuitka:WARNING:/usr/local/lib/python3.6/dist-packages/chevron/main.py:16: Cannot find 'metadata' in package 'chevron' as absolute import (tried metadata).
Nuitka:WARNING:/usr/local/lib/python3.6/dist-packages/yaml/cyaml.py:5: Cannot find '_yaml' in package 'yaml' as absolute import (tried _yaml).
Nuitka:WARNING:Unresolved '__import__' call at '/usr/local/lib/python3.6/dist-packages/yaml/constructor.py:505' may require use of '--include-plugin-directory' or '--include-plugin-files'.
Nuitka:WARNING:Unresolved '__import__' call at '/usr/local/lib/python3.6/dist-packages/yaml/constructor.py:521' may require use of '--include-plugin-directory' or '--include-plugin-files'.
```
I installed the packages via PIP and I would like to use the packages from PIP. Yes, I can use the `--include-plugin-directory` or the `--include-plugin-files`. However, I do not want to download the source code for both Chevron and PyYAML. Instead I would like to provide a list with `filepath:linenumber: import statement` entries instead.

For example:
```
[chevron/main.py](https://github.com/noahmorrison/chevron/blob/master/chevron/main.py)
try:
    from .renderer import render
    from .metadata import version
except (ValueError, SystemError):  # python 2
    from renderer import render
    from metadata import version
```

```
imports.txt
chevron/main.py:15: from .renderer import render
chevron/main.py:16: from .metadata import version

$ python3 -m nuitka --import-file imports.txt mycli.py
```
So, basically the idea is to tell Nutika what to overwrite, but we don't have to change the source code.

Python and Nutika version:

~~~
python3 -m nuitka --version
0.6.0.5
Python: 3.6.6 (default, Sep 12 2018, 18:26:19) 
Executable: /usr/bin/python3
OS: Linux - Ubuntu
Arch: x86_64
~~~

**Comment by kayhayen** [comment]
Is this only to disable the warnings, or should it actually impact behavior?

**Comment by kayhayen** [comment]
Without example or explanation there is no point in having it open.
